docs/section10/10-3.md

```diff
@@ -0,0 +1,47 @@
+# Airside systems – Rule 10-3  
+**Schema Version:** 0.0.13    
+**Mandatory Rule:** True     
+**Rule ID:** 10-3  
+ 
+**Rule Description:** For systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+
+**Rule Assertion:** B-RMR = expected value                                            
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. The hvac system in B-RMR serves a computer room.  
+
+**Function Call:**  
+
+1. get_zones_computer_rooms()  
+2. get_list_hvac_systems_associated_with_zone()  
+
+**Rule Logic:**  
+**Applicability Check 1:**  
+- Call function to get dictonary of zones that include computer rooms and associated square footage in B_RMR: `zones_that_include_comp_rooms_dict_b = = get_zones_computer_rooms(B_RMR)`
+- Get list of the zones that include computer rooms: `zones_that_include_comp_rooms_list_b = zones_that_include_comp_rooms_dict_b.keys()`
+- For each zone_b in zones_that_include_comp_rooms_list_b: `For zone_b in zones_that_include_comp_rooms_list_b:`
+    - Reset does_zone_have_dehumification boolean variable: `does_zone_have_dehumification = FALSE`
+    - Reset has_terminal_heating boolean variable: `has_terminal_heating = FALSE`
+    - Get the hvac system associated with the zone (should only be one, this RDS assumes one HVAC system): `hvac_b = get_list_hvac_systems_associated_with_zone(B_RMR, zone_b.id)`
+    - Check if the zone's maximum_humidity_setpoint_schedule equals Null: `if zone_b.maximum_humidity_setpoint_schedule == Null: does_zone_have_dehumification = FALSE`
+    - Else (i.e. if the zone's maximum_humidity_setpoint_schedule does NOT equal Null): `Else:`
+        - Loop though all elements in the schedule and check if the value equals 1 (convert to an 8760 schedule with a function if needed): `For x in Range(len(zone_b.maximum_humidity_setpoint_schedule.hourly_values))`
+            - Check if the value does not equal one, if it does not then set boolean variable to TRUE: `if zone_b.maximum_humidity_setpoint_schedule.hourly_values[x] != 1: does_zone_have_dehumification = TRUE`
```

Comment:
@claperle and @mkarpman I am unclear on what 'maximum humidity setpoint schedule' is meant to contain. Attached screenshot. The description says its meant to contain the min or max relative humidity setpoint and then the notes say it would be a fraction between 0 and 1. Which one is correct?
If it is a fraction, do we have a place where we specifying the absolute humidity setpoint which is getting adjusted by this fraction schedule. 


<img width="513" alt="image" src="https://user-images.githubusercontent.com/8323384/167563286-46d31c20-5527-4bd2-a091-683f04144de9.png">

Reply:
@supriyagoel I think there is no conflict because relative humidity is a percentage needed for saturation at the same temperature.

Reply:
@mkarpman, makes sense. then I guess we just need to add units as 'unitless' and clarify that this is a %. I'll create a RMD issue for @JasonGlazer